lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -63,6 +64,10 @@
         valid=specs.is_in("urllib", "curl", None),
         default=None
     ),
+    transport_timeout=dict(
+        map=lambda val: None if val == "None" else int(val),
```

Comment:
I'd prefer to see ``val.lower() == "none"`` - we use that other places. I'm fine if you don't want to switch this though.